package.json

```diff
@@ -45,6 +46,8 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongodb": "^3.5.5"
+    "json-server": "^0.16.1",
+    "mongodb": "^3.5.5",
+    "react-query": "^1.2.9"
```

Comment:
`react-query` is a client dependency (`client/package.json`)

Reply:
oh no, how could this happen... now I understand why Heroku had a bug with the package JSON 😅🙈